UnityProject/Assets/Scripts/Health/Objects/Armor.cs

```diff
@@ -24,6 +24,7 @@ public class Armor
 	[Range(-100,100)] public float Acid;
 	[Range(-100,100)] public float Magic;
 	[Range(-100,100)] public float Bio;
+	[Range(0, 100)] public float Anomaly;
```

Comment:
Why not follow the pattern of allowing weakness to be set?

Reply:
The way anomaly armour works right now is that its a percentae change of negating the effects, 100 - 100% of negating the effect, 0 - 0% chance, doesn't really make sense to have negatives here unless i am to change how the armour works, but im not sure what I would change it too.